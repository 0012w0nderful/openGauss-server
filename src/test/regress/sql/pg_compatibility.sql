-- pg compatibility case
drop database if exists pg_type_databse;
create database pg_type_databse dbcompatibility 'PG';

\c pg_type_databse
create table d_format_test(a varchar(10) not null);
insert into d_format_test values('');



-- concat test
select concat(null,'','','') is null;

select concat('','') is null;

select ''::int;