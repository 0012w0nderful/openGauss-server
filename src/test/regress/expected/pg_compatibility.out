-- pg compatibility case
drop database if exists pg_type_databse;
NOTICE:  database "pg_type_databse" does not exist, skipping
create database pg_type_databse dbcompatibility 'PG';

\c pg_type_databse
create table d_format_test(a varchar(10) not null);
insert into d_format_test values('');



-- concat test
select concat(null,'','','') is null;
 ?column?
----------
 f
(1 row)


select concat('','') is null;
 ?column?
----------
 f
(1 row)


select ''::int;
ERROR:  invalid input syntax for integer: ""
LINE 1: select ''::int;
               ^
CONTEXT:  referenced column: int4